src/components/Sidebar/Sidebar.js

```diff
@@ -71,12 +73,16 @@ export const Sidebar = () => {
       </NavLink>
       <hr />
 
-      <li onClick={logoutHandler}>
-        <MdExitToApp size={25} />
-        <span>Log Out</span>
-      </li>
+      {isAuth && (
+        <>
+          <li onClick={logoutHandler}>
+            <MdExitToApp size={25} />
+            <span>Log Out</span>
+          </li>
```

Comment:
semantic is not correct, `li` is a list item so it should have a parent, `ul` or `ol`, also in this file you have put 
`<NavLink to="/history" className="navlink">
        <li>
          <MdHistory size={25} />
          <span>History</span>
        </li>
      </NavLink>`

so `NavLink` should be the not be outside of the `li`, the order should be `ul` -> `li` -> `NavLink`

Reply:
will fix it. Thank you